static/js/break.js

```diff
@@ -10,22 +10,22 @@ timer.addEventListener('breakDone', breakDoneListener);
 
 function startBreak(){
   isBreak = true;
+  totaltimectr = true;
```

Comment:
Please follow existing coding format such as variable name. In this case `totalTimeControl` is a better choice.